### DISCUSSION TITLE
LF2 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#141] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF2.html

Author: @kathypinzon 
Reviewer: @AbbyANoble

**Comment by AbbyANoble** [comment]
In 3.2.1, it would be helpful if the graph had some gridlines to make the points easier to see. And/or labels counting by 1 instead of 2.

**Reply by AbbyANoble** [reply]
In [Activity 3.2.4(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-slope-int-form-2),  [Activity 3.2.8(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-point-slope-form-2) , and [Activity 3.2.11(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-line-eq-2) too.

**Comment by AbbyANoble** [comment]
In [Activity 3.2.1(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-need-y-2), the answer choices for A and B in parts (a) and (b) are not in math mode.

**Comment by AbbyANoble** [comment]
In [Activity 3.2.1(e)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-need-y-6), I'd recommend two separate questions. "What is the same about these two lines? What is different?"

**Comment by AbbyANoble** [comment]
[Remark 3.2.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#LF2-3-3) -- Maybe a slight change in wording -- "It is not enough to just know one piece of information **to define** a line" or "It is not enough to just know one piece of information to **to determine** line"

**Comment by AbbyANoble** [comment]
[Activity 3.2.5(c)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-2-points-4), instead of not possible, I'd say it's often hard to draw an accurate enough graph to determine the y-intercept.

**Reply by AbbyANoble** [reply]
I might also add a hint on this one to get them started?

**Comment by AbbyANoble** [comment]
Typo in [Remark 3.2.6](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#LF2-3-7) -- form **of** the equation of a line

**Comment by AbbyANoble** [comment]
[Activity 3.2.8](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-point-slope-form), an equation instead of the equation.

**Comment by AbbyANoble** [comment]
In [Activity 3.2.8(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-point-slope-form-2), could you do another correct choice using the other point you've pointed out?

**Reply by kathypinzon** [reply]
i added it as a 5th option, but should we include the y-0= or just y=

**Reply by AbbyANoble** [reply]
I could go either way? Part of me likes the y-0 so it really looks like point-slope, but part of me likes things simplified.

**Comment by AbbyANoble** [comment]
[Activity 3.2.8(c)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-point-slope-form-4), C should be "y+4=..." not "y-4=...". Could also include another correct choice using the other point.

**Comment by AbbyANoble** [comment]
[Activity 3.2.9(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#activity-2-points-revisit-2), and equation, not the

**Comment by AbbyANoble** [comment]
[Definition 3.2.14](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF2.html#def-hor-vert-lines) reads like it's saying k has a slope of zero and h has an undefined slope. I'd suggest 

A horizontal line has a slope of zero and is written in the form y=k where k is a constant. A vertical line has an undefined slope and is written in the form x=h where h is a constant.
